src/strategy/values/InvalidTargetValue.cpp

```diff
@@ -27,6 +27,7 @@ bool InvalidTargetValue::Calculate()
                target->isFeared() ||
                target->HasUnitState(UNIT_STATE_ISOLATED) ||
                target->IsFriendlyTo(bot) ||
+               !AttackersValue::IsValidTarget(target, bot) ||
```

Comment:
@liyunfan1223 
This modification will cause the bots no longer attack some units with rti7 (Such as the Bone Spike of Lord Marrowgar and the Kor'kron Battle-Mage on the battle Gunship), Although we can attack them through attach commands. Is there any way to optimize it?